docs/external-classes.json

```diff
@@ -1,98 +1,50 @@
 [{
   "name": "Google\\Auth\\",
-  "uri": "https://github.com/google/google-auth-library-php/blob/master/src/%s.php"
+  "uri": "https://github.com/google/google-auth-library-php/blob/master/src/%s.html"
 }, {
   "name": "Google\\GAX\\",
-  "uri": "https://github.com/googleapis/gax-php/tree/master/src/%s.php"
+  "uri": "https://googleapis.github.io/gax-php/master/Google/GAX/%s.html"
```

Comment:
After some thought, I think that we have three cases: `x.y.z` (specific-version), `^1.x` (GA), and `^0.x` (pre-GA), which actually work out very similar in practice. For a specific-version dependency, we just point to the docs for that specific version. For a `^1.x` dependency, we want to point to the latest `1.*` release. In practice, this is the auth library, and pointing at master makes sense - any changes will be driven by our teams and included in a release, and we can coordinate a 2.0 release if that is ever necessary. For a `^0.x` dependency, we want to point to the latest `0.x.*` release. In practice, this is the GAX and proto-client libraries, which we control. Whenever we release a new version, it will be because we want to use it in google-cloud-php, and we can control the GAX doc site redirection so that it gets updated only when google-cloud-php takes a dependency on the latest release (see https://github.com/googleapis/gax-php/blob/gh-pages/index.html). Once GAX goes GA, the situation will be similar to the auth library, and we will want to point to the latest release all of the time. Similarly we can also coordinate a 2.0 release with google-cloud-php so that the docs are not out of sync if that is ever necessary.

So, the upshot is that I think leaving the docs with the current behaviour (redirecting to "latest" for GAX, pointing to master for auth) might be the way to go.

WDYT?